tests/test_training.py

```diff
@@ -52,30 +86,40 @@ def gpt_tiny_cfg(conf_path: str = 'scripts/train/yamls/pretrain/mpt-125m.yaml'):
                      not torch.cuda.is_available(),
                      reason='testing with cuda requires GPU')),
 ])
-@pytest.mark.parametrize('logit_scale', [None, 0.036, 'inv_sqrt_d_model'])
-def test_train(device: str, logit_scale: Optional[Union[float, str]]):
-    if not os.path.isdir('./my-copy-c4/val'):
-        pytest.xfail('c4 dataset not set up as expected')
-
-    warnings.filterwarnings(
-        action='ignore',
-        category=DeprecationWarning,
-        message=
-        "Using the 'grad_clip_norm' field in Trainer is deprecated. Please usethe GradientClipping Algorithm in composer.algorithms.gradient_clipping."
-    )
-
-    test_cfg = gpt_tiny_cfg(
-        conf_path='scripts/train/yamls/pretrain/mpt-125m.yaml')
-    test_cfg.eval_subset_num_batches = 2
-    if logit_scale:
-        test_cfg.model.logit_scale = logit_scale
+def test_train(device: str):
+    create_c4_dataset_xsmall()
```

Comment:
should probably `reproducibility.seed_all` here to ensure fixed randomness.

Reply:
main seeds_all right away, but I still agree, its not a bad idea to do it here.

Reply:
this is tested in main so probably don't want to add here imo as it would be overwritten anyways